src/utils/DragAndDrop.js

```diff
@@ -332,13 +335,17 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
+            var ext = Path.extname(item.name).replace(/^\./, "").toLowerCase();
```

Comment:
Fair point. looking at the rest of the code there are also a fair number of spots where file type checks based on extension run through `Content.is[...](Path.extname(filename))`, does it make sense to extend Content here so that it has an `isArchive()` function that accepts .tar, .zip, etc?